be/src/olap/compaction.cpp

```diff
@@ -737,40 +666,12 @@ Status Compaction::do_inverted_index_compaction() {
         }
     }
 
-    std::vector<InvertedIndexFileInfo> all_inverted_index_file_info(dest_segment_num);
-    uint64_t inverted_index_file_size = 0;
-    for (int seg_id = 0; seg_id < dest_segment_num; ++seg_id) {
-        auto inverted_index_file_writer = inverted_index_file_writers[seg_id].get();
-        if (Status st = inverted_index_file_writer->close(); !st.ok()) {
-            status = Status::Error<INVERTED_INDEX_COMPACTION_ERROR>(st.msg());
-        } else {
-            inverted_index_file_size += inverted_index_file_writer->get_index_file_total_size();
-            inverted_index_file_size -= compacted_idx_file_size[seg_id];
-        }
-        all_inverted_index_file_info[seg_id] = inverted_index_file_writer->get_index_file_info();
-    }
     // check index compaction status. If status is not ok, we should return error and end this compaction round.
     if (!status.ok()) {
         return status;
     }
-
-    // index compaction should update total disk size and index disk size
-    _output_rowset->rowset_meta()->set_data_disk_size(_output_rowset->data_disk_size() +
-                                                      inverted_index_file_size);
-    _output_rowset->rowset_meta()->set_total_disk_size(_output_rowset->data_disk_size() +
-                                                       inverted_index_file_size);
-    _output_rowset->rowset_meta()->set_index_disk_size(_output_rowset->index_disk_size() +
-                                                       inverted_index_file_size);
-
-    _output_rowset->rowset_meta()->update_inverted_index_files_info(all_inverted_index_file_info);
-    COUNTER_UPDATE(_output_rowset_data_size_counter, _output_rowset->data_disk_size());
-
     LOG(INFO) << "succeed to do index compaction"
-              << ". tablet=" << _tablet->tablet_id() << ", input row number=" << _input_row_num
-              << ", output row number=" << _output_rowset->num_rows()
-              << ", input_rowset_size=" << _input_rowsets_size
-              << ", output_rowset_size=" << _output_rowset->data_disk_size()
-              << ", inverted index file size=" << inverted_index_file_size
+              << ". tablet=" << _tablet->tablet_id()
```

Comment:
Why all other info erased?

Reply:
Other info cannot be obtained here. After the compaction is fully successful, relevant info will be printed.